initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -516,7 +518,7 @@ startCmd() {
         elif [ ${RCSTART} -eq 2 ]; then
           COMPONENT_HAS_STARTED=1
         else
-          if [ ${COMPONENT_HAS_STARTED} -eq 1 ]; then
+          if [ ${COMPONENT_HAS_STARTED} -eq 1 ] || [ ${RCSTART} -eq 5 ]; then
```

Comment:
RCSTART is check_status result.. I can only see it returns 0-4, what is this test against 5?